build.psm1

```diff
@@ -602,7 +602,11 @@ Fix steps:
 
     if ($Environment.IsWindows) {
         ## need RCEdit to modify the binaries embedded resources
-        if (-not (Test-Path "~/.rcedit/rcedit-x64.exe")) {
+        $rcedit = "~/.rcedit/rcedit-x64.exe"
+        if (-not (Test-Path -Type Leaf $rcedit)) {
+            $rcedit = Get-Command rcedit-x64 -ErrorAction SilentlyContinue | Select-Object -First 1 | % Name
```

Comment:
Should it be `Get-Command rcedit-x64.exe -CommandType Application -ErrorAction SilentlyContinue `?

Reply:
Yes, that's better.